.circleci/config.yml

```diff
@@ -107,32 +107,32 @@ jobs:
           name: Clone benchmarks
           command: |
             cd /home/cbgeo/project/build
-            git clone https://github.com/cb-geo/mpm-benchmarks.git benchmarks
+            git clone https://github.com/geomechanics/mpm-examples.git benchmarks
       # Benchmarks
       - run:
           name: 2D sliding block
           command: |
             cd /home/cbgeo/project/build
             pip3 install tables --user
             export OMP_NUM_THREADS=1
-            mpirun -n 4 ./mpm -f ./benchmarks/2d/sliding_block_inclined_boundary/
-            python3 ./benchmarks/2d/sliding_block_inclined_boundary/test_benchmark.py
+            mpirun -n 4 ./mpm -f ./benchmarks/explicit_single_phase/validation/sliding_block_inclined_boundary/
+            python3 ./benchmarks/explicit_single_phase/validation/sliding_block_inclined_boundary/test_benchmark.py
       # Stress
       - run:
           name: Uniaxial stress
           command: |
             cd /home/cbgeo/project/build
-            ./mpm -f ./benchmarks/2d/uniaxial_stress/
-            python3 ./benchmarks/2d/uniaxial_stress/test_benchmark.py
+            ./mpm -f ./benchmarks/explicit_single_phase/validation/uniaxial_stress/
+            python3 ./benchmarks/explicit_single_phase/validation/uniaxial_stress/test_benchmark.py
```

Comment:
Wait, not actually approving this. All of the input files should be a subdirectory called "inputs"

For example:
```
python3 ./benchmarks/explicit_single_phase/validation/uniaxial_stress/inputs/test_benchmark.py
```

Reply:
PR opened https://github.com/geomechanics/mpm-examples/pull/5

Reply:
OK! Does it look ok now?